app/helpers/users_helper.rb

```diff
@@ -8,4 +8,13 @@ def user_avatar(user, classes)
       @avatar_user = image_tag("/avatar.jpg", alt: user.name, class: classes)
     end
   end
+
+  def admin_users(current_user,user)
+    if current_user == user && user.admin?
+    elsif user.admin?
+      link_to "Remove as Admin", admin_user_path(user), method: :patch, class: 'btn btn-outline-danger btn-block'
+    else
+      link_to "Set as Admin", admin_user_path(user), method: :patch, class: 'btn btn-outline-info btn-block'
+    end
```

Comment:
@ivanraybuglosa You can use structure like this as well:
![image](https://user-images.githubusercontent.com/20507565/46327277-63839680-c633-11e8-9630-afa271d4f80b.png)